### DISCUSSION TITLE
Gitlab support ?

### DISCUSSION THREAD

**Discussion by vanlongme** [coollabsio/coolify#104] [Feature Requests]
Gitlab also provide awesome all-in-one platform for self-host including repositories, container registry, gitlab-ci,... and so on, do you support Gitlab on the furture or else ?

**Comment by andrasbacsai** [comment]
The new version (v2) is supporting GitLab, hosted (gitlab.com), and self-hosted as well!

**Comment by andrasbacsai** [comment]
As v2 is out, it supports GitLab. :)
